src/kakarot/instructions/environmental_information.cairo

```diff
@@ -549,7 +549,7 @@ namespace EnvironmentalInformation {
         bitwise_ptr: BitwiseBuiltin*,
     }(ctx: model.ExecutionContext*) -> model.ExecutionContext* {
         // Get return data size.
-        let return_data_size = Helpers.to_uint256(ctx.sub_context.return_data_len);
+        let return_data_size = Helpers.to_uint256(ctx.sub_context.return_info.len);
```

Comment:
here again, i face the confusion between the words size and len, and i don't really know what is what
though i'm playing a bit dumb, but we may be able to find ubiquitous language to guide us through this hardship

Reply:
i started with just using the terms as they are used in the call opcodes and was planning on clarifying the names once everything compiled, passed the pytest stuff, and the inciting ef-test case that inspired this ask for a refactor